### ISSUE TITLE
Bytecode 12 support

### ISSUE THREAD

**Issue by Grossley** [krzys-h/UndertaleModTool#445]
Bytecode 13 support is here, which is great. And I found a bytecode 12 game. It's not inconceivable that there would be more games on this bytecode format. And I was able to get it to load and save with only a few minor changes to UndertaleModTool on the experimental branch.

**Comment by Miepee** [comment]
Do you happen to have links to the bytecode 12 games?

**Comment by Grossley** [comment]
> Do you happen to have links/names to the bytecode 12 games?

Yes I do, but I would need some time to find them again. They are free and publicly available.
